### DISCUSSION TITLE
Various Artist albums seem to import with wrong album_artist

### DISCUSSION THREAD

**Discussion by Andrew-McGee** [ampache/ampache#2892] [General]
I have imported a few compilation albums lately and noticed the album artist is being set to one of the song artists even though all the song tags have an album artist = 'Various Artists'.

I will test a couple more and if I see the same behaviour I will raise it as an issue. This is a recent change - previously the behaviour was as expected.

**Comment by lachlan-00** [comment]
possibly realated to my commits searching by mbid, i'll double check mine too.

There is a fall back fallback that i think could be the culprit

take this out of `src/Repository/Model/Artist.php`
![image](https://user-images.githubusercontent.com/1305249/115368812-cd241a00-a20a-11eb-9ce6-7a41294b0714.png)

**Reply by Andrew-McGee** [reply]
It could be something on my side. I did a couple recently and they imported fine. I am re-checking my tags in case I messed it up.
